titus-server-master/src/test/java/com/netflix/titus/master/integration/v3/scenario/JobAsserts.java

```diff
@@ -38,15 +38,16 @@
         return task -> task.getStatus().getState() == expectedState;
     }
 
-    public static Predicate<TaskExecutorHolder> containerWithResources(ContainerResources containerResources) {
+    public static Predicate<TaskExecutorHolder> containerWithResources(ContainerResources containerResources, int diskMbMin) {
         return taskExecutorHolder -> {
             if (taskExecutorHolder.getTaskCPUs() != containerResources.getCpu()) {
                 return false;
             }
             if (taskExecutorHolder.getTaskMem() != containerResources.getMemoryMB()) {
                 return false;
             }
-            if (taskExecutorHolder.getTaskDisk() != containerResources.getDiskMB()) {
+            int diskMB = Math.max(containerResources.getDiskMB(), diskMbMin);
```

Comment:
we should probably change the units to be all correct cased instead of camel casing them as `Mb` != `MB`.